src/variables/variable_units_invested_available.jl

```diff
@@ -57,7 +61,6 @@ function add_variable_units_invested_available!(m::Model)
         lb=constant(0),
         int=units_invested_available_int,
         fix_value=fix_units_invested_available,
-        internal_fix_value=internal_fix_units_invested_available,
         initial_value=initial_units_invested_available,
```

Comment:
This needs to be `initial_value=_initial_units_invested_available` - note the sneaky leading underscore - to use the function defined above in line 47. Otherwise the unit investment variable is not fixed to zero at the beginning of the horizon. The same should be donde for storages and connections in the respective files. This should fix Benders tests.

To explain, the function `_initial_units_invested_available` defined on line 47 ensures that the initial value is either the value defined by the user via the `initial_units_invested_available` parameter, or zero if no value defined.

Nice work @DillonJ @tarskul this seems to be coming along nicely. Sorry for the merge issues, most of them must be my fault...

Reply:
Thanks  @manuelma it wasn't actually that difficult to resolve the conflicts and I think we all equally contributed to the issues :-) I've made that fix - let's see how the tests go!